docs/image/index.html

```diff
@@ -76,7 +76,62 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
             </li>
           </ul>
         </aside>
-        <article>image</article>
+        <article>
+          <h3 class="heading-3">Images</h3>
+          <div class="component-description">
+            <p class="justify">
+              Images are inseparable part of Websites. You can use the class
+              <code class="inline-code">.image-responsive</code> for responsive
+              images, <code class="inline-code">.image-square</code> for square
+              type image, and <code class="inline-code">.image-round</code> for
+              a circular image.
+            </p>
+          </div>
+          <h4 class="heading-4">Responsive Image</h4>
+          <div class="component-example flex flex-center">
+            <img
+              src="https://images.unsplash.com/photo-1494500764479-0c8f2919a3d8?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"
+              loading="lazy"
+              class="image-responsive"
+              alt="Otago, New-Zealand"
+            />
+          </div>
+          <div class="code-example">
+            <pre><code class="html">
+<img src="url comes here" class="image-responsive" alt="alt text" />
+						</code></pre>
+          </div>
+
+          <h4 class="heading-4">Square Image</h4>
+          <div class="component-example flex flex-center">
+            <img
+              src="https://images.unsplash.com/photo-1494500764479-0c8f2919a3d8?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"
+              loading="lazy"
+              class="image-square"
+              alt="Otago, New-Zealand"
+            />
+          </div>
+          <div class="code-example">
+            <pre><code class="html">
+<img src="url comes here" class="image-square" alt="alt text" />
+						</code></pre>
+          </div>
+
```

Comment:
Indentation not proper

Reply:
That is because I use highlight JS to show code examples and it doesnot show well if I follow indentation